app/controllers/Application.scala

```diff
@@ -101,6 +101,26 @@ class Application(
     }
   }
 
+  def contributionsLandingUS(title: String, id: String, js: String, newDesigns: String): Action[AnyContent] = CachedAction() { implicit request =>
+    if (newDesigns == "circles-garnett") {
```

Comment:
I've seen multiple PRs with this condition. Is the value `circles-garnett` defined as a constant somewhere?

Reply:
Hmm, could define a `val` for this scoped to the `Application` class?

Reply:
Sounds good.